packages/wonder-blocks-grid/components/grid.js

```diff
@@ -103,6 +100,10 @@ export default class Grid extends React.Component<
         this.watchHandlers = {};
 
         for (const size of VALID_GRID_SIZES) {
+            if (!spec[size]) {
+                continue;
+            }
```

Comment:
Would this happen if a grid spec only defined a single size, e.g. `large`?  Are there situations in which it makes sense to only define a single size?

Reply:
Or if there are cases where there are two specs specified (like in the case of the modal - there is no medium size).